### DISCUSSION TITLE
Firewall and Virus detection.

### DISCUSSION THREAD

**Discussion by 0v3rlurd** [Khopa/dcs_liberation#1823] [Q&A] [answered]
Hello.

My virus protection keeps on getting rid of the main exe. Anyone else run into this?

![image](https://user-images.githubusercontent.com/95365343/144259587-86ec7fe7-b459-405a-a319-39a4ba6064b3.png)

**Comment by Starfire13** [comment, marked as answer]
Add an exception in your antivirus settings.
